### PR TITLE
Changed primary commands to be consistent with the toolbar

### PR DESCRIPTION
**Pull request by yaichenbaum**

### CODE REVIEW

File: Files/Helpers/ContextFlyoutItemHelper.cs [hunk 1]
```diff
@@ -547,15 +545,14 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseItemMenuItems(BaseLayo
                     GlyphFontFamilyName = "CustomGlyph",
                     Command = commandsViewModel.OpenDirectoryInNewTabCommand,
                     ShowItem = selectedItems.Count < 5 && selectedItems.All(i => i.PrimaryItemAttribute == Windows.Storage.StorageItemTypes.Folder),
-                    IsPrimary = true,
+
```
Comment (yaichenbaum) [inline Files/Helpers/ContextFlyoutItemHelper.cs:548]:
```suggestion
```

**Comment by winston-de** [comment]
should the create folder with selection option be moved down?
image

**Comment by yaichenbaum** [comment]
> should the create folder with selection option be moved down?
> image

@winston-de I made this change in the latest commit.

**Review by winston-de — APPROVED** [review]
lgtm

**Pull request merged**